static/js/map/map.settings.js

```diff
@@ -203,6 +204,18 @@ function initSettingsSidebar() {
             Store.set('filterPokemonById', this.checked)
         })
 
+        $('#exclude-nearby-cell-switch').on('change', function () {
+            settings.excludeNearbyCells = this.checked
+            updateMap({ loadAllPokemon: true })
```

Comment:
When toggling on this setting, the Pokémon won't disappear until you reload the page. When toggling off, they will appear right away. 
For this work in both cases,  you will also need to use the `updatePokemons` function.